SecondSub.py

```diff
@@ -0,0 +1,105 @@
+def blend(pic1, pic2):
+
+  # Assumes the pictures are the same size
```

Comment:
Use a doc-string to describe the function. Use comments to clarify code and/or explain why it is written the way it is.